### DISCUSSION TITLE
Prebuilt Radio Groups in 1.0@next

### DISCUSSION THREAD

**Discussion by sidis405** [vidstack/player#939] [Q&A] [answered]
Hi all

I pulled in a Vuejs (Inertiajs) project the new version of Vidstack and started customising the Tailwind layout as per this example [https://github.com/vidstack/examples/blob/main/player/vue/tailwind-css/src/components/layouts/VideoLayout.vue](https://github.com/vidstack/examples/blob/main/player/vue/tailwind-css/src/components/layouts/VideoLayout.vue)

In the example, in the settings menu we've got only the `CaptionSubmenu` which is fine.
Using `media-quality-radio-group` i added the qualities submenu as well.

I can't find though the web components to add the audio tracks and the playback rate to the settings menu (equivalent of media-quality-radio-group and media-captions-radio-group essentially).

Am i missing something obvious? Thank you in advance

**Comment by mihar-22** [comment, marked as answer]
Hey @sidis405!

The other radio groups would be `media-audio-radio-group` and `media-speed-radio-group`.

**Reply by sidis405** [reply]
@mihar-22 hi and thank you for the prompt response. 

I figured but autocomplete wasn't pulling them up and writing that anyway yielded nothing. 

But I will try first thing tomorrow morning. 

Thank you again and keep up the excellent work:)

**Reply by mihar-22** [reply]
The IDE sometimes bugs out, I just checked and the types are there. Give it another shot tomorrow and see how you go :)

**Reply by sidis405** [reply]
And you were absolutely right :)
